test/specs/create-journey-tests.e2e.js

```diff
@@ -196,5 +197,21 @@ describe('Create journey tests', () => {
 
       await expect(await SecretsPage.secretCell(keyName)).toExist()
     })
+
+    it('Should be able to view list of test-suites with new test-suite listed', async () => {
+      // eslint-disable-next-line wdio/no-pause
+      await browser.pause(1000) // Wait for the secret to be created
```

Comment:
if this bites, in the future you can simply wait for something to be on the page?